docs/02-about-us/working-groups/readme.md

```diff
@@ -0,0 +1,9 @@
+# Working Groups
+
+CivicActions maintains [a set of working groups](working-groups) that nurture specific 
+initiatives within the organization.
+
+# Group Listing
+
+* [Documentation](documentation.md)
```

Comment:
ok i just saw the /docs/readme.md below, i think you want your link to go there?